recipe/meta.yaml

```
@@ -1,17 +1,19 @@
-{% set version = "7.5.0" %}
-{% set libgfortran_version = "4.0.0" %}
-{% set max_libgfortran_version = "5.0.0.a0" %}
+{% if gfortran_version == "7.5.0" %}
+{% set libgfortran_major_version = "4" %}
+{% else %}
+{% set libgfortran_major_version = "5" %}
+{% endif %}
 
 package:
   name: gfortran_osx-64
-  version: {{ version }}
+  version: {{ gfortran_version }}
 
 build:
-  number: 5
+  number: 6
   skip: True  # [win]
   run_exports:
     strong:
-      - libgfortran >={{ libgfortran_version }},<{{ max_libgfortran_version }}
+      - libgfortran{{ libgfortran_major_version }} >={{ gfortran_version }}
```

To enforce mutual exclusivity.
```suggestion
      - libgfortran
      - libgfortran{{ libgfortran_major_version }} >={{ gfortran_version }}
```